backend/chat_messages/models/message.py

```diff
@@ -58,7 +58,7 @@ class Meta:
         verbose_name_plural = "Chats"
         ordering = ["-last_message_at", "-created_at"]
 
-    def __str__(self):
+    def __DEPRECEATED_str__(self):
```

Comment:
@positiveimpact 
Please check your changes here. A new __str__ was added while leaving the other one. The class had 2 __str__ methods.

Reply:
Thanks! Removed the old str method.